CHANGELOG.md

```diff
@@ -30,11 +30,11 @@ master
 * Clear lazy map after resolving Tilt templates (#2132)
 * Fix ignore of I18n files (#2143)
 * Fix redirect destination lookup (#2140) 
-* Add LD-JSON to MinifyJavascript content types allowed to be compressed (#2138)
+* Add LD-JSON to MinifyJavaScript content types allowed to be compressed (#2138)
```

Comment:
This might not be a changable thing, as it may be a public api:
https://www.rubydoc.info/github/middleman/middleman/Middleman/Extensions/MinifyJavascript

Reply:
Should be fine, if changed everywhere.